chcemvediet/apps/inforequests/cron.py

```diff
@@ -199,6 +201,43 @@ def close_inforequests():
             trace = unicode(traceback.format_exc(), u'utf-8')
             cron_logger.error(msg.format(inforequest, trace))
 
+@cron_job(run_at_times=settings.CRON_IMPORTANT_MAINTENANCE_TIMES)
+def publish_inforequests():
+    if not settings.AUTOPUBLISH_INFOREQUESTS:
```

Comment:
@martinmacko47 Alebo tento stav osetrim v `settings.commpon.py`, tak ze cronjob vobec nepridam do zoznamu `CRON_CLASSES`?

Reply:
@viliambalaz Radsej `if` nechaj vo vnutri `publish_inforequests`, lebo v `settings.commpon.py` by vyhadzovanie `publish_inforequests` bolo komplikovane, kedze `AUTOPUBLISH_INFOREQUESTS` sa meni po nastaveni `CRON_CLASSES`. Dalo by sa to, ale bolo by to matuce imho.